src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ChangeDirectionDueDateHandler.java

```diff
@@ -88,7 +88,10 @@ public boolean canHandle(
                                     asylumCase.read(AsylumCaseFieldDefinition.DIRECTION_EDIT_DATE_DUE, String.class).orElse(""),
                                     dateProvider.now().toString(),
                                     idValue.getValue().getTag(),
-                                    appendPreviousDates(idValue.getValue().getPreviousDates(), idValue.getValue().getDateDue(), idValue.getValue().getDateSent())
+                                    appendPreviousDates(idValue.getValue().getPreviousDates(), idValue.getValue().getDateDue(), idValue.getValue().getDateSent()),
+                                        Collections.emptyList(),
+                                        UUID.randomUUID().toString(),
+                                        callback.getEvent().toString()
```

Comment:
we should use same uniqueId and directionType from previous direction:
```
idValue.getValue().getUniqueId(), idValue.getValue().getDirectionType()
```
only direction due date change

Reply:
Thanks Marcin. I changed in in the other place but missed this. It has been corrected now.